### PR TITLE
Add @floatingstatic as pack maintainer

### PR DESCRIPTION
**Pull request by floatingstatic**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# when someone opens a pull request.
+
+# This is base configuration. These owners could review the
+# whole file in this repository.
+* @StackStorm-Exchange/tsc @floatingstatic
+
+# CI configuration files should be reviewed by specific owners
+# who are more responsible for ensuring the quality of this pack
+# or orchestrate StackStorm-Exchanges.
+.circleci/**  @StackStorm-Exchange/tsc
```
Comment (arm4b) [inline .github/CODEOWNERS]: We can remove the CircleCI part, as it's not used anymore in Exchange

**Review by arm4b — APPROVED** [review]
Thanks for taking care of this pack :+1: 

Left one comment, also changelog and pack version bump seem unnecesarry. We can ignore the CI failures asking to update the Changelog for this PR

File: .github/CODEOWNERS (hunk 1 above)
Comment (floatingstatic) [reply .github/CODEOWNERS]: Ah, yeah I was wondering about that. Its still in many pack repos I looked at. Removing!

**Comment by arm4b** [comment]
Thanks @floatingstatic!
You should get an invite link from Github to maintain this repo.

**Pull request merged**